### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_Summary_ScreenReader:The name of the page is not announced.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9348]
[Check out Accessibility Insights](https://accessibilityinsights.io/)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#VoiceOver;#MAS1.3.1;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Screen reader: Voiceover
Mac OS:10.15.3
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Launch screen reader as Voiceover 
8. Select Configuration profile --> Update Azure settings --> Update Cluster settings --> Update Service settings & hit Next.

**Actual:**
When the page is loaded the "alert" message is announced and the name of the screen in the display is not announced.

**Expected:**
The screen name in display should be announced.

**User Impact:**
Screen reader users will not know which page they have landed on.

**Attachment for Reference :**
![A11y_SQLAzureDataStudio_SQLServerBigDataCluster_Summary_ScreenReaderMac](https://user-images.githubusercontent.com/24580570/75327881-1ac12300-58a3-11ea-8b3b-72a2a4dafb3e.png)

**Comment by alanrenmsft** [comment]
if we have 2 alert elements, the second one is not getting announced.

**Comment by sspatil123** [comment]
As "SQL Server Big Data Cluster" feature has been removed
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
